Create README (#7)
webex/component-adapter-interfaces: README.md

> <h1 align='center' style='border-bottom: none;'>Webex Component Adapters</h1>

Reviewer: ```suggestion
<h1 align='center' style='border-bottom: none;'>Webex Component Adapter Interfaces</h1>
```

> <h3 align='center'>Library of Adapters for Webex components</h3>

Reviewer: ```suggestion
<h3 align='center'>Library of Adapter Interfaces for Webex components</h3>
```

<p align='center'>
  <a href='https://circleci.com/gh/webex/component-adapter-interfaces'>
    <img alt='CircleCI' src='https://circleci.com/gh/webex/component-adapter-interfaces.svg?style=shield'>
  </a>
  <a href='https://www.npmjs.com/package/@webex/component-adapter-interfaces'>
    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/component-adapter-interfaces?label=npm%40latest'>
  </a>
  <a href='#badge'>
    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
  </a>
  <a href='https://github.com/webex/component-adapter-interfaces/blob/master/package.json#L28'>
    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
  </a>
</p>

> **Webex Component Adapters** is a set of adapters...

Reviewer: ```suggestion
**Webex Component Adapter Interfaces** are a set of adapter interfaces which are utilized when implementing a <link needed> Webex Component Adapter.
```

> **Webex Component Adapter Interfaces** is a set of interfaces...

Reviewer 2: ```suggestion
**Webex Component Adapter Interfaces** is a set of interfaces that Webex Components (<= make this a link to the repo) use as a contract on how to obtain data.
```

> **Webex Component Adapter Interfaces** is a set of interfaces that Webex Components (<= make this a link to the repo) use as a contract on how to obtain data.

Reviewer: ```suggestion
**Webex Component Adapter Interfaces** is a set of interfaces that [Webex Components](https://github.com/webex/components#adapters) use as a contract on how to obtain data.
```

## Table of Contents

- [Table of Contents](#table-of-contents)
- [Install](#install)
- [Usage](#usage)
- [Contributing](#contributing)
  - [Issues](#issues)
  - [Contributing Code](#contributing-code)
  - [Release Process](#release-process)
    - [Commit linter](#commit-linter)
- [License](#license)
- [Support](#support)
- [Team](#team)

## Install

```bash

> npm install --save @webex/component-adapters-interfaces

Reviewer: ```suggestion
npm install --save @webex/component-adapter-interfaces
```

```

## Usage

An example of extending the `RoomsAdapter` for your datasource:

```js
import {RoomsAdapter} from '@webex/components-adapter-interfaces';

export default class MyRoomsAdapter extends RoomsAdapter {
  getRoom(ID) {
   // ... my implemenation
  }
}
```

_Happy Coding!_

## Contributing

> We'd love for you to contribute to our source code and to make **Webex Component Adapters** even better than it is today! Here are some [guidelines](https://github.com/webex/components/blob/master/CONTRIBUTING.md) that we'd like you to follow.

Reviewer: `Webex Component Adapter Interfaces`

By default `semantic-release` uses the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) and triggers release and publishing based on the following rules:

| Commit                             | Release type  |
| ---------------------------------- | ------------- |
| Commit with type `BREAKING CHANGE` | Major release |
| Commit with type `feat`            | Minor release |
| Commit with type `fix`             | Patch release |
| Commit with type `perf`            | Patch release |

#### Commit linter

We are using [commitlint](https://github.com/conventional-changelog/commitlint) to lintify the commit messages.
Please make sure to choose the appropriate commit [type](https://github.com/webex/components/blob/master/CONTRIBUTING.md#type), [scope](https://github.com/webex/components/blob/master/CONTRIBUTING.md#scope) and [subject](https://github.com/webex/components/blob/master/CONTRIBUTING.md#scope).

## License

[MIT License](https://opensource.org/licenses/MIT)

## Support

For more developer resources, tutorials and support, visit the Webex developer portal, https://developer.webex.com.

## Team

> | [![Adam Weeks](https://github.com/adamweeks.png?size=100)](https://github.com/adamweeks) | [![Arash Koushkebaghi](https://github.com/akoushke.png?size=100)](https://github.com/akoushke) | [![Lalli Flores](https://github.com/lalli-flores.png?size=100)](https://github.com/lalli-flores) | [![Timothy Scheuering](https://github.com/InteractiveTimmy.png?size=100)](https://github.com/InteractiveTimmy) | [![David Hoff](https://github.com/harborhoffer.png?size=100)](https://github.com/harborhoffer) |

Reviewer 2: uhm, do you wanna add yourself to the team list? 😅

Reviewer 3: Copy paste this:

Reviewer 3: ## Team

| [![Adam Weeks](https://github.com/adamweeks.png?size=100)](https://github.com/adamweeks) | [![Arash Koushkebaghi](https://github.com/akoushke.png?size=100)](https://github.com/akoushke) | [![Lalli Flores](https://github.com/lalli-flores.png?size=100)](https://github.com/lalli-flores) | [![Timothy Scheuering](https://github.com/InteractiveTimmy.png?size=100)](https://github.com/InteractiveTimmy) | [![David Hoff](https://github.com/harborhoffer.png?size=100)](https://github.com/harborhoffer) | [![Taymoor Khan](https://github.com/taymoork2.png?size=100)](https://github.com/taymoork2) |
| ---------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ |
| [Adam Weeks](https://github.com/adamweeks)                                               | [Arash Koushkebaghi](https://github.com/akoushke)                                              | [Lalli Flores](https://github.com/lalli-flores)                                                  | [Timothy Scheuering](https://github.com/InteractiveTimmy)                                                      | [David Hoff](https://github.com/harborhoffer)                                                  | [Taymoor Khan](https://github.com/taymoork2)                                               |

Reviewer 3: make sure it looks good though. Preview it.

<p align='center'>
  <a href='https://circleci.com/gh/webex/component-adapter-interfaces'>
    <img alt='CircleCI' src='https://circleci.com/gh/webex/component-adapter-interfaces.svg?style=shield'>
  </a>
  <a href='https://www.npmjs.com/package/@webex/component-adapter-interfaces'>
    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/component-adapter-interfaces?label=npm%40latest'>
  </a>
  <a href='#badge'>
    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
  </a>
  <a href='https://github.com/webex/component-adapter-interfaces/blob/master/package.json#L28'>
    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
  </a>
</p>

**Webex Component Adapters** is a set of adapters...

## Table of Contents

- [Table of Contents](#table-of-contents)
- [Install](#install)
- [Usage](#usage)
- [Contributing](#contributing)
  - [Issues](#issues)
  - [Contributing Code](#contributing-code)
  - [Release Process](#release-process)
    - [Commit linter](#commit-linter)
- [License](#license)
- [Support](#support)
- [Team](#team)

## Install

```bash

> npm install --save @webex/component-adapters

Reviewer: ```suggestion
npm install --save @webex/component-adapters-interfaces
```

```

## Usage

> _Happy Coding!_

Reviewer: Webex Component Adapter Interfaces are utilized when implementing a (link needed) Webex Component Adapter. 

An example of extending the RoomsAdapter for your datasource:
 
```suggestion
import {RoomsAdapter} from '@webex/components-adapter-interfaces';

export default class MyRoomsAdapter extends RoomsAdapter {
  getRoom(ID) {
   // ... my implemenation
  }
}
```
